cmd/statusd/utils.go

```diff
@@ -1414,6 +1418,29 @@ func testJailFunctionCall(t *testing.T) bool {
 	return true
 }
 
+func testNodeOffline(t *testing.T) bool {
+
+	StopNode()
+	//time to sync
+	time.Sleep(5 * time.Second)
```

Comment:
Unless we can guarantee some means of knowing the total time of sync, we can only guess here. Can we expose some channel which gets closed once syncing is done to resolve this type of approach?